app/views/pages/api_release_notes.html.haml

```diff
@@ -9,6 +9,126 @@
     Github issues
   for bugs or other problems.
 
+%section.api-documentation
+  %hr
+  %h2 28th February 2019
```

Comment:
As there are 3 `update` for Feb 28th, should these be grouped as a single date?
Similar to 3rd September which had two deprecation warnings and an update
#not_a_blocker

Reply:
👍 will do